app/src/main/java/org/fossasia/openevent/general/event/Event.kt

```diff
@@ -78,5 +89,58 @@ data class Event(
     @ColumnInfo(index = true)
     @Relationship("event-sub-topic", resolve = true)
     var eventSubTopic: EventSubTopic? = null
-
 )
+
+object DataBindingAdapter {
```

Comment:
Why is this in `Events.kt`?

Reply:
i will be moving it into Utils once the pr is complete for the enitre project

Reply:
Why in Events.kt in the first place? This needs to be in its own separate file

Reply:
there were binding adapters specific to this model only so i was thinking to keep them in this file only and some common binding adapetrs like visibllity in a common singleton class